### PR TITLE
Adds different static files

### PR DESCRIPTION
**Pull request by sobolevn**
Changes:
1. Adds `editorconfig` to enforce custom rules on all text files
2. Updates `gitignore` to cover more `elixir` specific cases
3. Adds `CONTRIBUTING` to show how to contribute
4. Updates `README` with different minor changes
5. Adds `CHANGELOG` to follow semantic versioning

### CODE REVIEW

**Comment by sobolevn** [comment]
The next step will include:

- [ ] tests
- [ ] coverage
- [ ] credo linting
- [ ] dialixyr
- [ ] travis
- [ ] badges

**Comment by sobolevn** [comment]
Logo is not shown yet, since it relies on master branch.

File: .gitignore [hunk 1]
```diff
@@ -1,17 +1,9 @@
-# The directory Mix will write compiled artifacts to.
```
Comment (sntran) [inline .gitignore]: Is there a reason to remove comments on this file? They add clarification to newcomers, and they don't cost anything.

Comment (sobolevn) [inline .gitignore]: Yeap, will do.

**Comment by sobolevn** [comment]
Done! Thanks for the review.

**Comment by sntran** [comment]
Thank you!

**Pull request merged**